### PR TITLE
people/dcallega: Add Davide’s personal page

### PR DESCRIPTION
**Pull request by dcallega**
Minimal content for my page

### CODE REVIEW

File: content/people/dcallega.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+# Page front matter
```
Comment (igorburago) [inline content/people/dcallega.md]: Please remove the comments from the metadata in the front matter.

**Review by igorburago — CHANGES_REQUESTED** [review]

File: content/people/dcallega.md [hunk 2]
```diff
@@ -1,93 +1,27 @@
...
   - Distributed Machine Learning in Heterogeneous Computer Networks
   - Reinforcement Learning and Semantic Maps
   - Unmanned Aerial Vehicles
   - Edge Computing
-# A list of acquired degrees, in reversed chronological order,
-# each described by three fields: official degree title, name of
-# the university, and year of graduation.
 education:
   - degree: M.S. in Computer Engineering
     institution: Università degli Studi di Padova
```
Comment (igorburago) [inline content/people/dcallega.md]: Let’s use English names of institutions, i.e., “University of Padova, Italy” in this case.

File: content/people/dcallega.md [hunk 3]
```diff
@@ -1,93 +1,27 @@
...
     year: 2016
   - degree: B.S. in Information Engineering
     institution: Università degli Studi di Padova
```
Comment (igorburago) [inline content/people/dcallega.md]: Same here.

File: content/people/dcallega.md [hunk 4]
```diff
@@ -1,93 +1,27 @@
...
-# Filename of an optional avatar photo to be shown in the people
-# directory page. Should be square. If missing, the portrait photo is
-# used instead. The file should be put in the same directory as the
-# portrait photo.
 badge: dcallega-badge.jpg
-# A list of free-form topics constituting your main research
-# interests. Keep each short and to the point; do not put a period
-# at the end.
 interests:
   - Distributed Machine Learning in Heterogeneous Computer Networks
```
Comment (igorburago) [inline content/people/dcallega.md]: Let’s not use title case in research interests; please don’t capitalize every word.

File: content/people/dcallega.md [hunk 5]
```diff
@@ -1,93 +1,27 @@
...
     institution: Università degli Studi di Padova
     year: 2016
   - degree: B.S. in Information Engineering
     institution: Università degli Studi di Padova
     year: 2013
-# Optional email address.
 email: dcallega@uci.edu
-# Optional URL of your personal website (outside of the lab website).
-# Do not forget to include the protocol part, e.g., 'http://' or 'https://'.
 website: https://www.linkedin.com/in/davide-callegaro/
```
Comment (igorburago) [inline content/people/dcallega.md]: Website entry syntax [has changed][commit] to support (multiple) links with
titles. Please change this to:

    websites:
      - title: LinkedIn
        url: https://www.linkedin.com/in/davide-callegaro/

(In general, the `title` field is optional.)

[commit]: https://github.com/uci-iasl/website/commit/46182f701b9a2d85ced05103826c242ea412af3c

**Review by igorburago — CHANGES_REQUESTED** [review]

File: content/people/dcallega.md (hunk 5 above)
Comment (dcallega) [reply content/people/dcallega.md]: done

File: content/people/dcallega.md [hunk 6]
```diff
@@ -6,19 +6,21 @@ yearjoined: 2016
 portrait: dcallega.jpg
 badge: dcallega-badge.jpg
 interests:
-  - Distributed Machine Learning in Heterogeneous Computer Networks
-  - Reinforcement Learning and Semantic Maps
-  - Unmanned Aerial Vehicles
-  - Edge Computing
+  - distributed machine learning in heterogeneous computer networks
```
Comment (igorburago) [inline content/people/dcallega.md]: Keep the very first letter capitalized, of course.
E.g.: “Reinforcement learning and semantic maps”.

The point of my comment in the previous review was:
Just do not use title case; capitalize the words the same way you
would normally in a sentence.

File: content/people/dcallega.md [hunk 7]
```diff
@@ -6,19 +6,21 @@ yearjoined: 2016
...
+  - reinforcement learning and semantic maps
+  - unmanned aerial vehicles
+  - edge computing
 education:
   - degree: M.S. in Computer Engineering
-    institution: Università degli Studi di Padova
+    institution: University of Padova
```
Comment (igorburago) [inline content/people/dcallega.md]: Please add the country: “University of Padova, Italy”.

**Review by igorburago — CHANGES_REQUESTED** [review]

**Review by igorburago — APPROVED** [review]
LGTM

**Pull request merged**